README.md

```diff
@@ -129,7 +129,7 @@ PHP: SDK|Every Wednesday at 10:30 PT|[Google Doc](https://docs.google.com/docume
 Python: SDK|Every Thursday at 09:00 PT|[Google Doc](https://docs.google.com/document/d/1CIMGoIOZ-c3-igzbd6_Pnxx1SjAkjwqoYSUWxPY8XIs/edit)|[Slack](https://cloud-native.slack.com/archives/C01PD4HUVBL)|
 Ruby: SIG|Every Tuesday at 09:00 PT|[Google Doc](https://docs.google.com/document/d/1D15bO8o340sQm2CVZiukEJuCO_XMMHKPuTznoEhyFqE/edit)|[GitHub Discussions](https://github.com/open-telemetry/opentelemetry-ruby/discussions)|
 Rust: SDK|Every other week on Tuesday alternating between 07:00 and 15:00 PT|[Google Doc](https://docs.google.com/document/d/1tGKuCsSnyT2McDncVJrMgg74_z8V06riWZa0Sr79I_4)|[Gitter](https://gitter.im/open-telemetry/opentelemetry-rust)|
-Specification: General|Every Tuesday at 08:00 PT|[Google Doc](https://docs.google.com/document/d/1pdvPeKjA8v8w_fGKAN68JjWBmVJtPCpqdi9IZrd6eEo/edit)|[Slack](https://cloud-native.slack.com/archives/C01N7PP1THC)
+Specification: General|Every Tuesday at 08:00 PT, Every Tuesday at 16:00 PT|[Google Doc](https://docs.google.com/document/d/1pdvPeKjA8v8w_fGKAN68JjWBmVJtPCpqdi9IZrd6eEo/edit)|[Slack](https://cloud-native.slack.com/archives/C01N7PP1THC)
```

Comment:
@arminru want to get your input here. During the TC meeting we were thinking about the "OpenTelemetry Specs SIG: APAC" and probably rename the 8AM one to "OpenTelemetry Specs SIG: EMEA/NA/LATAM". I want to get your perspective if we can just call both meetings "OpenTelemetry Specs SIG" (which implies: anyone from whatever geo-location are welcomed to join, if the time makes sense to them).

![image](https://user-images.githubusercontent.com/17327289/162488206-6bfb22e4-97ed-4f71-920d-8249b13feedf.png)